tests/phpunit/TemplateTest.php

```diff
@@ -1122,7 +1122,7 @@ public function testJstorSICIEncoded() {
   public function testOverwriteBlanks() {
     $text = '{{cite journal|url=http://www.jstor.org/stable/1234560|jstor=}}';
     $expanded = $this->process_citation($text);
-    $this->assertEquals('{{cite journal|jstor=1234560}}', $expanded->parsed_text());
+    $this->assertEquals(1, substr_count($expanded->parsed_text(), 'jstor'));
```

Comment:
But won't this return success if the text is not expanded, by virtue of the `jstor` in the URL?

Reply:
This test is not to see if it expanded or not, but if we remember to not leave the blank jstor= too.  The code at one point did that